docs/ReleaseNotes.md

```diff
@@ -1,50 +1,93 @@
 # IPDK Kubernetes* Infrastructure Offload Release Notes
 
-## What's New in This Release
+## Releases
 
-### IPDK 23.07 K8s-Infra-Offload Component
+### IPDK 23.07
 
-- This is the release of K8s-Infra-Offload recipe for ES2K and DPDK targets.
-This is the first release that supports both these targets.
+- This is the first release of K8s-Infra-Offload recipe that supports ES2K and DPDK targets.
 
 ### Highlights
 
-#### DPDK and ES2K targets
-- Support for Kubernetes Container Network Interface (CNI) to enable pods to
-  send/receive traffic.
-- Intra Node L3 Forwarding to enable pod to pod communication, on the same node,
-  via CNI interfaces.
+#### ES2K target
+
+- Support for Kubernetes Container Network Interface (CNI) to deploy pods and
+  enable pod-to-pod connectivity on a P4 target using hardware device interfaces.
+- Use of internal gateway with dummy MAC to enable layer-3 connectivity on the same node.
+- Support for SRIOV and dynamic Subfunctions on ES2K.
+  Subfunction is a lightweight function that has a parent PCI function on which it is
+  deployed. It is created and deployed in a unit of 1. Unlike SRIOV VFs, a subfunction
+  doesn't require its own PCI virtual function. A subfunction communicates with the
+  hardware through the parent PCI function.
+- Infra Manager build support on ARM cores.
 
 #### DPDK target
-- Service Load Balancing within the node to allow multiple pods on same node to
+
+- Support for internal gateway with dummy MAC to enable layer-3 connectivity on the
+  same node.
+- service of type=ClusterIP support.
+  Service Load Balancing within the node to allow multiple pods on same node to
   act as end points providing any application service.
 - Bi-directional Auto Learning and Flow Pinning (a.k.a Connection Tracking),
   used with load balancing, to allow consistent end point pod selection, once it
   has been selected for the first packet.
 - DNS service provided by Core DNS pods to other pods.
 
-### Security Domain
+### Common Changes
 
-This release does not support multi-tenant or multi-node deployments. At
-present, the underlying IPDK networking recipe needs to be run on bare metal
-on host CPU cores. The entire node, used for deployment, is assumed to be a
-trusted zone. However, gRPC/gNMI channels for communications are still
-secured using TLS. For that, the user should create the TLS keys and
-certificates and install them as per the steps given in the README.
+- Makefile target to support tls-secrets and certificate generation
+- Automatated build & integration test on each commit
+- Felix integration and communication with Infrastructure Offload Components.
+- Addition of DB to store state information.
+- Support for Fedora33, Fedora37
```

Comment:
Should we claim support for F37 when it has not been tested?

Reply:
SQA ran on Fedora37. We haven't. Wil check with Deepa once again.